edgelet/docker-rs/src/models/host_config.rs

```diff
@@ -168,12 +168,12 @@ pub struct HostConfig {
     /// Specification for mounts to be added to the container.
     #[serde(rename = "Mounts", skip_serializing_if = "Option::is_none")]
     mounts: Option<Vec<crate::models::Mount>>,
-    // /// A list of kernel capabilities to add to the container.
-    // #[serde(rename = "CapAdd", skip_serializing_if = "Option::is_none")]
-    // cap_add: Option<Vec<String>>,
-    // /// A list of kernel capabilities to drop from the container.
-    // #[serde(rename = "CapDrop", skip_serializing_if = "Option::is_none")]
-    // cap_drop: Option<Vec<String>>,
+    /// A list of kernel capabilities to add to the container.
+    #[serde(rename = "CapAdd", skip_serializing_if = "Option::is_none")]
+    cap_add: Option<Vec<String>>,
```

Comment:
I think it would be simpler for `cap_add` and `cap_drop` to just be `Vec<String>`. You would just have to change to `skip_serializing if = "Vec::is_empty"` and make the default value `vec![]`.

Reply:
This was the auto-generated code, I just uncommented the fields I was using. I didn't see a reason to change the values. I agree that it's simpler that way though. I'll go ahead and change it.